src/components/activity-completion/completion-page-content.tsx

```diff
@@ -165,10 +167,25 @@ export const CompletionPageContent: React.FC<IProps> = (props) => {
           <div className={exitContainerClass} data-cy="exit-container">
             <h1>Summary of Work: <span className="activity-title">{activityTitle}</span></h1>
             <SummaryTable questionsStatus={progress.questionsStatus} />
-            {showStudentReport && <button className="button show-my-work" onClick={handleShowAnswers}><IconCompletion width={24} height={24} />Show My Work</button>}
-            {(!sequence || isLastActivityInSequence) && 
+              { showStudentReportButton
+                  ?
+                    <button
+                      className="button show-my-work"
+                      onClick={handleShowAnswers}>
+                        <IconCompletion width={24} height={24} />
+                        Show My Work
+                    </button>
+                  :
+                    <button
+                      className="button show-my-work disabled"
+                      disabled={true}>
+                        <IconCompletion width={24} height={24} />
+                        Show My Work
+                    </button>
+              }
+            {(!sequence || isLastActivityInSequence) &&
               <div className="exit-button">
-                <span>or</span>
+                { showStudentReportButton && <span>or</span> }
```

Comment:
Seems like the `or` should be shown all of the time now that the button is just disabled.